bessctl/conf/samples/vxlan.bess

```diff
@@ -36,33 +36,34 @@ Container Bob  |    (regular Ethernet port)    |
 
 import os
 import socket
+import struct
 
-def aton(ip):
-    return socket.inet_aton(ip)
+def atoh(ip):
+    return struct.unpack("!L", socket.inet_aton(ip))[0]
 
-os.system('docker pull ubuntu > /dev/null')
-os.system('docker run -i -d --net=none --name=vxlan_test ubuntu /bin/bash > /dev/null')
+os.system('sudo docker pull ubuntu > /dev/null')
+os.system('sudo docker run -i -d --net=none --name=vxlan_test ubuntu /bin/bash > /dev/null')
 
 # Alice live outside the container, wanting to talk to Bob in the container
 v_bob = VPort(ifname='eth_bob', docker='vxlan_test', ip_addrs=['172.16.100.33/28'])
 v_alice = VPort(ifname='eth_alice', ip_addrs=['10.0.10.2/24'], mac_addr='02:0a:0b:0c:0d:0e')
 
 # Alice -> Bob
-PortInc(port=v_alice) -> bpf::BPF():1 -> VXLANDecap() -> PortOut(port=v_bob)
+PortInc(port=v_alice.name) -> bpf::BPF():1 -> VXLANDecap() -> PortOut(port=v_bob.name)
```

Comment:
Originally when `Module` or `Port` instances appear as a parameter, they used to be implicitly converted to its name (string). Please see https://github.com/NetSys/bess/blob/c-legacy/bessctl/commands.py#L514